backend/Makefile

```diff
@@ -12,7 +12,10 @@ venv:
 install: venv
 	. $(VIRTUAL_ENV)/bin/activate && \
 	pip install --upgrade pip && \
-	pip install -r requirements.txt
+	pip install -r requirements.txt && \
+    wget -O /tmp/ifcopenshell_python.zip "https://s3.amazonaws.com/ifcopenshell-builds/ifcopenshell-python-311-v0.7.0-742a9ec-linux64.zip" && \

```

Comment:
Is it safe to assume 3.11? I normally do

    `python3 -c 'import sys;print("".join(map(str, sys.version_info[0:2])))'`

Reply:
The docker images start with a 3.11 image, therefore the intent is to "pin" to that version.